app/views/kudos/_kudos.html.erb

```diff
@@ -1,26 +1,26 @@
 <% # expects local variables kudos, guest_kudos_count, commentable %>
 <div id="kudos">
-  <% if kudos.length > 0 || guest_kudos_count > 0 %>
+  <% if kudos.any? || guest_kudos_count > 0 %>
```

Comment:
There seems to be some discussion of exists? and any? however the sql at least looks the same

Loading staging environment (Rails 5.1.3)
2.3.0 :001 > ActiveRecord::Base.logger = Logger.new(STDOUT)
 => #<Logger:0x000000087c0bc0 @progname=nil, @level=0, @default_formatter=#<Logger::Formatter:0x000000087c0b98 @datetime_format=nil>, @formatter=nil, @logdev=#<Logger::LogDevice:0x000000087c0af8 @shift_size=nil, @shift_age=nil, @filename=nil, @dev=#<IO:<STDOUT>>, @mon_owner=nil, @mon_count=0, @mon_mutex=#<Thread::Mutex:0x000000087c0aa8>>>
2.3.0 :002 > w=Work.find(128033)
D, [2018-06-27T19:05:57.370903 #23670] DEBUG -- :   Work Load (1.4ms)  SELECT  `works`.* FROM `works` WHERE `works`.`id` = 128033 LIMIT 1
D, [2018-06-27T19:05:57.376094 #23670] DEBUG -- :   ↳ (irb):2:in `irb_binding'
 => #<Work id: 128033, ....moderated_commenting_enabled: false>
2.3.0 :003 > w.kudos.any?
D, [2018-06-27T19:06:06.620124 #23670] DEBUG -- :   Kudo Exists (1.6ms)  SELECT  1 AS one FROM `kudos` WHERE `kudos`.`commentable_id` = 128033 AND `kudos`.`commentable_type` = 'Work' LIMIT 1
D, [2018-06-27T19:06:06.622624 #23670] DEBUG -- :   ↳ (irb):3:in `irb_binding'